src/js/profile/chart.js

```diff
@@ -464,11 +464,11 @@ export class Chart extends Observable {
                     this.setDownloadUrl();
                     this.vegaView.signal(`${filterName}Filter`, true)
                     this.vegaView.signal(`${filterName}FilterValue`, sf.value)
-                    this.appendDataToTable()
                 }
             });
         }
-        this.vegaView.run()
+        this.vegaView.run();
+        this.appendDataToTable();
```

Comment:
just noting while I see it - appendDataToTable doesn't sound idempotent, but the way it's being used, it sounds like it should be.

Is it safe to call it many times? Does it truncate the table for us? In that case "updateTable" would make more sense, rght?